go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kanisterio/kanister
 
-go 1.22.7
+go 1.23.2
```

Comment:
please separate this change. will this break dependent projects that use this as a library?

Reply:
no.  ive skipped this pr now for the prs in the chain.  the function was being used to wait after sending signals in the Signal API tests, but ive backported those to use Stdout and GetProcess.